internal/controller/postgrescluster/pgbackrest.go

```diff
@@ -2758,12 +2810,45 @@ func (r *Reconciler) reconcileRepos(ctx context.Context,
 	}
 
 	postgresCluster.Status.PGBackRest.Repos =
-		getRepoVolumeStatus(postgresCluster.Status.PGBackRest.Repos, repoVols, extConfigHashes,
-			replicaCreateRepo.Name)
+		getRepoVolumeStatus(postgresCluster.Status.PGBackRest.Repos, repoVols,
+			extConfigHashes, replicaCreateRepo.Name)
 
 	return replicaCreateRepo, utilerrors.NewAggregate(errors)
 }
 
+// +kubebuilder:rbac:groups="",resources="pods",verbs={list}
+
+// getRepoHostVolumeRequests gets the pgBackRest repo host volume request annotations
+// from the repo host Pod and stores them in the Postgres Cluster object's status.
+// If there is no repo host or there are no annotations, this function returns
+// without updating the status.
+func (r *Reconciler) getRepoHostVolumeRequests(ctx context.Context,
```

Comment:
i'd like this function name to say something about status 🤔 ⛏️

Reply:
Maybe `writeRepoVolumeSizeRequestStatus`? It's still a little long, but it's pretty descriptive. I don't think we need annotation here necessarily.

Reply:
I pushed that name.